app/models/mixins/miq_web_server_worker_mixin.rb

```diff
@@ -15,7 +16,7 @@ def self.binding_address
 
     def self.rails_server_command_line
       @rails_server_command_line ||= begin
-        "#{self.nice_prefix} #{Rails.root.join("bin", "rails")} server".freeze
+        "#{nice_prefix} #{Rails.root.join("bin", "rails")} server #{RAILS_SERVER}".freeze
```

Comment:
@matthewd If that works, I'm good with that... we can add that constant if we need it later.

I was under the impression you still had to tell rails server which webserver to use.